Dockerfile

```diff
@@ -120,6 +120,13 @@ ENV CASSANDRA_CLUSTER=false \
 # Configure search nodes. Empty string falls back to search being done on the nodes themselves.
 ENV PEGA_SEARCH_URL=
 
+# Configure hazelcast. By default, hazelcast runs in embedded mode.
+Env CLIENT_MODE=false \
+    ON_PEGA_K8S= \
+    CLUSTER_NAME= \
```

Comment:
Actually for all of these env vars please prefix them with HZ to make it clear they are related to hazelcast

Reply:
Sure, I will do this change.